src/pages/settings/Profile/ProfilePage.js

```diff
@@ -165,27 +173,22 @@ class ProfilePage extends Component {
         }
 
         // Check if the user has modified their avatar
-        if ((this.props.currentUserPersonalDetails.avatar !== this.state.avatar.uri) && this.state.isAvatarChanged) {
-            // If the user removed their profile photo, replace it accordingly with the default avatar
-            if (this.state.avatar.uri.includes('/images/avatars/avatar')) {
-                PersonalDetails.deleteAvatar(this.state.avatar.uri);
-            } else {
-                PersonalDetails.setAvatar(this.state.avatar);
-            }
+        if ((lodashGet(this.props.currentUserPersonalDetails, 'avatar') !== this.state.avatar.uri) && this.state.isAvatarChanged) {
+            PersonalDetails.setAvatar(this.state.avatar);
 
             // Reset the changed state
             this.setState({isAvatarChanged: false});
         }
 
-        PersonalDetails.setPersonalDetails({
-            firstName: this.state.firstName.trim(),
-            lastName: this.state.lastName.trim(),
-            pronouns: this.state.pronouns.trim(),
-            timezone: {
+        PersonalDetails.updateProfile(
+            this.state.firstName.trim(),
+            this.state.lastName.trim(),
+            this.state.pronouns.trim(),
+            {
                 automatic: this.state.isAutomaticTimezone,
                 selected: this.state.selectedTimezone,
             },
-        }, true);
+        );
```

Comment:
This tiny change gave rise to https://github.com/Expensify/App/issues/10215

Note: The main issue is opacity being incorrect, not the Growl being missing